backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -192,7 +194,14 @@ export class ContributionResolver {
       logger.error('user of the pending contribution and send user does not correspond')
       throw new Error('user of the pending contribution and send user does not correspond')
     }
-
+    if (
+      contributionToUpdate.contributionStatus !== ContributionStatus.IN_PROGRESS &&
```

Comment:
use `contributionSattus.in(valueA, valueB)` ?

Reply:
This does not work, contributionStatus is a String and can't use the in method